cmd/run/run.go

```diff
@@ -389,6 +390,8 @@ func (s *ServerContext) Run(ctx context.Context, config *serverconfig.Config) er
 				requestid.NewStreamingInterceptor(),
 				validator.StreamServerInterceptor(),
 				grpc_ctxtags.StreamServerInterceptor(),
+				recovery.StreamPanicInterceptor(s.Logger),
+				recovery.StreamPanicInterceptor(s.Logger),
```

Comment:
You only need to register it once. You can drop this duplicate registration.

Reply:
Done